precompiles/distribution/query_test.go

```diff
@@ -93,6 +93,8 @@ func (s *PrecompileTestSuite) TestValidatorDistributionInfo() {
 		{
 			"success",
 			func() []interface{} {
+				// FIXME this could be broken
```

Comment:
What does this mean?

Reply:
Many interfaces change on the sdk v0.50. This includes the `x/staking` module.
For this case in particular, previously (in v0.47.x), the `GetOperator()` returned a `sdk.ValAddress`. Now, it returns a `string`. This change breaks many parts of the code